config-scalecube-configuration-service/src/test/java/io/scalecube/config/service/BrokerData.java

```diff
@@ -0,0 +1,57 @@
+package io.scalecube.config.service;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class BrokerData {
+
+  private String BrokerID;
+  private ApiKey[] APIKeys;
```

Comment:
Maybe `apiKeys` ?

Reply:
This is the data structure in production